src/webgpu/api/validation/resource_usages/texture/in_pass_encoder.spec.ts

```diff
@@ -1185,6 +1207,13 @@ g.test('validation_scope,same_draw_or_dispatch')
   });
 
 g.test('validation_scope,different_draws_or_dispatches')
+  .desc(
+    `
+    Test the texture usages with the different draw or dispatch on the same pass that has two bind
+    groups. A validation error should be generated even though the pass draws or dispatches per each
+    bindGroup.
```

Comment:
Upon review of this test, I think it's broken. I will open a change after this lands to fix some issues here.

```suggestion
    Test usage scope validation with two conflicting bind groups used in two different draw/dispatch calls.

    TODO: This test is failing validation due to a different thing that it intends to test.
```